themes/mlibrary_new/exhibit-builder/exhibits/browse.php

```diff
@@ -3,7 +3,7 @@
   * Copyright (c) 2016, Regents of the University of Michigan.
   * All rights reserved. See LICENSE.txt for details.
   */
-
+set_theme_option('display_header','0');
```

Comment:
I strongly discourage calling `set_theme_option()` here.  It triggers a database update, which is going to cause a lot of churn for mysql.

Reply:
We do not want to display the banner image on the browse page. The header file is the file that will display the banner image. Setting this flag will allow us to check the header file to display the banner image or not. Other suggestions are welcome.

Reply:
You should only call `set_theme_option()` in response to a POST request, and from a controller handling that request.  Themes should not have side effects that alter the database.

From what I can tell, you're using the theme options like global variables with the lifetime of the current request, and this may have unintended consequences:
- Other legitimate uses of set_theme_option can get clobbered by these concurrent requests.
- The churn in mysql data can become untenable for our mysql servers.
- It introduces a bottleneck in serving pages because these updates lock table rows.

Ideally, the controller should be passing the view all the information it needs to render a page.  Realistically, I understand that that doesn't happen, but the controller is the first place to look for options on where or how to indicate whether the template should render that header.  Depending on how Zend handles things, this may not pan out, or it may be feasible through hooks on the controller.

Another option is to do as I suggested in my comment in the `header.php` diff, and create a filter on `theme_options`.  These are applied after the database is queried, and thus do not cause a database update.

Yet another option is to explicitly use a global variable, because that's how you're treating the theme option for `display_header`.  You may find it easier to pull from `$GLOBALS`, like `$GLOBALS['display_header']`.